# Definition of a box item for the boxpick component

# Optional
string[] grasp_uuids

# Required. The pose of the box item
geometry_msgs/Pose pose

# Required
string pose_frame

# Required
Rectangle rectangle

# Required
time timestamp

# Required
string type

# Required
string uuid
